checks/raw/security_policy.go

```diff
@@ -70,7 +81,24 @@ func SecurityPolicy(c *checker.CheckRequest) (checker.SecurityPolicyData, error)
 	}
 
 	// Return raw results.
-	return checker.SecurityPolicyData{Files: data.files}, nil
+	if len(data.files) > 0 {
+		for idx := range data.files {
+			filePattern := data.files[idx].File.Path
+			// undo path.Join in isSecurityPolicyFile
+			if data.files[idx].File.Type == checker.FileTypeURL {
+				filePattern = strings.Replace(data.files[idx].File.Path, data.uri+"/", "", 1)
```

Comment:
I have removed (and tested) needing the actual URI in the path.

Reply:
wdym?

Reply:
In the original code if the security policy file was not in the actual repo but in the repo's org `.github` folder - the `File.Type` would be set to `checker.FileTypeURL` and the URI would be prepended/joined to the found file name. In the new security policy which looks at the content, when trying to `fileparser.OnMatchingFileContentDo` the method would fail because doing that joining (in the original code) was incorrect--so as a temporary solution, I removed that which was joined so I could get the contents properly. HTH

Reply:
that is *incorrect* as a parameter to `fileparser.OnMatchingFileContentDo`

Reply:
I need to put that back the way it was... yes, I need to remove the `uri` portion of the path when getting the contents of the file using `OnMatchingFileContentDo` - but the `uri` portion needs to remain on the path for the `raw` output--when I did that work, I was not aware of the `raw` json output capability, so that part has to be restored. I will do that tonight. 

current official `raw` json reports `path` for an organization security policy as:

```
        "securityPolicies": [
            {
                "path": "github.com/gorilla/.github/SECURITY.md"
            }
```

where the new code on the same site reports `path` without the `uri` as (partial json here):

```
                ...}
                ],
                "path": "SECURITY.md"
```

my bad :-(